src/redturtle/prenotazioni/content/prenotazioni_folder.py

```diff
@@ -363,7 +374,7 @@ def get_options():
     )
 
     notBeforeDays = schema.Int(
-        default=2,
+        default=0,
```

Comment:
Why did this default change?

Reply:
bcs the written tests are not considering that the default is 2. we can change tests or we can change the default. So as tests are rappresentign the expected behaviour i changed the default